numba/unicode_support.py

```diff
@@ -207,12 +207,10 @@ def _PyUnicode_IsDecimalDigit(ch):
 
 @register_jitable
 def _PyUnicode_ToDigit(ch):
-    raise NotImplementedError
-
-
-@register_jitable
-def _PyUnicode_IsDigit(ch):
-    raise NotImplementedError
+    ctype = _PyUnicode_gettyperecord(ch)
```

Comment:
This needs a github source reference with specific SHA referring to the `ToDigit` implementation.